packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -71,6 +75,9 @@ const runSessionExpiryKey = "Fluid.GarbageCollection.RunSessionExpiry";
 const disableSessionExpiryKey = "Fluid.GarbageCollection.DisableSessionExpiry";
 // Feature gate key to log error messages if GC reference validation fails.
 const logUnknownOutboundReferencesKey = "Fluid.GarbageCollection.LogUnknownOutboundReferences";
+// Feature gate key to write the gc blob as a handle if the data is the same.
+const trackGCBlobStateKey = "Fluid.GarbageCollection.TrackGCBlobStateKey";
+const trackGCBlobStateVersionKey = "Fluid.GarbageCollection.TrackGCBlobStateVersionKey";
```

Comment:
Two things:
* Drop "Key" from both these new strings (keep it in the variable name only)
* Consider a naming convention like `Fluid.GarbageCollection.MinVersion.TrackGCBlobState` to make it clear the two settings are tied together  One is the value and the other is the min version where the setting is valid or whatever.

Reply:
Maybe 
`Fluid.GarbageCollection.TrackGCBlobState` and 
`Fluid.GarbageCollection.TrackGCBlobState.MinVersion`?